packages/svelte/src/runtime/ssr.js

```diff
@@ -10,11 +10,11 @@ export {
 	SvelteComponentTyped
 } from './index.js';
 
-/** @returns {void} */
```

Comment:
i'm not sure why these comments were removed

Reply:
The return void is deduced automatically,
and I removed it because other parts of the code didn't add these type annotations, but if it was a change to specify that nothing should be returned,
this change is should be roll back.